cmd/tesla-control/commands.go

```diff
@@ -491,6 +491,80 @@ var commands = map[string]*Command{
 			return car.ScheduleCharging(ctx, false, 0*time.Hour)
 		},
 	},
+    "departure-schedule": &Command{
```

Comment:
*nit*: indentation seems to be incorrect here

Reply:
Fixed